source/geometry_model/ellipsoidal_chunk.cc

```diff
@@ -65,20 +59,14 @@ namespace aspect
     EllipsoidalChunk<dim>::EllipsoidalChunkGeometry::set_initial_values(double para_semi_major_axis_a,

```

Comment:
By the way, why is the function called `set_initial_values`? "Initial" means something at the beginning of time. Would `set_geometry_parameters` or similar be a better name?